py3status/modules/do_not_disturb.py

```diff
@@ -42,6 +46,8 @@ class Py3status:
     format = '{state}'
     notification_manager = 'dunst'
     refresh_interval = 0.25
+    signal_off = 'SIGTERM'
+    signal_on = 'SIGUSR1'
```

Comment:
Since these are dunst-specific, I wonder if the parameter names should have `dunst_` prefix.

Reply:
Sure, this seems like a good idea, I'll change it.